src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -182,9 +183,14 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
     links: props => <ScrollablePromo {...props} />,
     mpu: props =>
       isAdsEnabled ? <MpuContainer {...props} slotType="mpu" /> : null,
-    wsoj: props => (
-      <CpsRecommendations {...props} items={recommendationsData} />
-    ),
+    wsoj: props =>
+      CpsVaried({
+        experimentID: null,
```

Comment:
The ArticlePage component is routed through Belfrage only, so it will receive MVT headers. Does the A/A test value need applied here instead?